app/components/hiring_staff/inline_components/vacancies_count_heading_component.rb

```diff
@@ -0,0 +1,12 @@
+class HiringStaff::InlineComponents::VacanciesCountHeadingComponent < ViewComponent::Base
+  def initialize(vacancies: vacancies, selected_type: selected_type)
+    @vacancies = vacancies
+    @selected_type = selected_type
+  end
+
+  def call
+    t("jobs.#{@selected_type}_jobs_with_count",
+      count: @vacancies.count,
+      jobs: @vacancies.count == 1 ? t('jobs.job') : t('jobs.job').pluralize)
```

Comment:
In fact if you don't use the translation file to do it, this component probably needs a test